test/xpu/xpu_test_utils.py

```diff
@@ -514,28 +515,30 @@ def replace_opinfo_device(self, info: OpInfo):
             info.decorators = tuple(decorator_xpu)
         skip_xpu = []
         replaced = False
-        for skip in info.skips:
-            if type(skip) == DecorateInfo:
-                if skip.device_type == "cuda":
-                    skip_xpu.append(decorator)
-                    skip.device_type = "xpu"
+        if hasattr(info, "skips"):
+            for skip in info.skips:
+                if type(skip) == DecorateInfo:
+                    if skip.device_type == "cuda":
+                        skip_xpu.append(decorator)
+                        skip.device_type = "xpu"
+                        replaced = True
+                    else:
+                        skip_xpu.append(skip)
+                elif self.only_cuda_fn == skip:
+                    skip_xpu.append(common_device_type.onlyCUDA)
                     replaced = True
-                else:
-                    skip_xpu.append(skip)
-            elif self.only_cuda_fn == skip:
-                skip_xpu.append(common_device_type.onlyCUDA)
-                replaced = True
         if replaced:
             info.skips = tuple(skip_xpu)
 
     def adjust_db(self, db):
         for opinfo in db:
-            if opinfo.name not in _xpu_computation_op_list:
-                opinfo.dtypesIfXPU = opinfo.dtypes
-            else:
-                backward_dtypes = set(opinfo.backward_dtypesIfCUDA)
-                backward_dtypes.add(bfloat16)
-                opinfo.backward_dtypes = tuple(backward_dtypes)
+            if not isinstance(opinfo, ModuleInfo):
```

Comment:
You are saying ops in `ModuleInfo` are not listed in our supported list? So,
1. What operators are included in `ModuleInfo`?
2. Looks no operator in `ModuleInfo` is supported by us?